memo_wise.gemspec

```diff
@@ -26,13 +26,10 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = Gem::Requirement.new(">= 2.5.0")
 
   # Specify which files should be added to the gem when it is released.
-  # The `git ls-files -z` loads the files in the RubyGem that have been added
-  # into git.
-  spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject do |f|
-      f.match(%r{^(test|spec|features)/})
-    end
-  end
+  spec.files = Dir.glob(
+    '{CHANGELOG.md,LICENSE.txt,README.md,lib/**/*.rb}',
+    File::FNM_DOTMATCH
```

Comment:
Why do we need `FNM_DOTMATCH`? Wouldn't it be the same without it since none of these files begin with `.`?

Reply:
Thanks, it is not needed, it is also wrong for this use case. I've used it somewhere else to exclude directories (`.`, `..`) but this is not the case